src/react-apps/applications/service-development/src/sharedResources/appRelease/get/getAppReleasesSagas.ts

```diff
@@ -2,15 +2,269 @@ import { AxiosError } from 'axios';
 import { delay, SagaIterator } from 'redux-saga';
 import { call, fork, race, take, takeLatest } from 'redux-saga/effects';
 import { checkIfAxiosError } from '../../../../../shared/src/utils/networking';
-import { get } from '../../../utils/networking';
-import { releasesUrlGet } from '../../../utils/urlHelper';
+// import { get } from '../../../utils/networking';
+// import { releasesUrlGet } from '../../../utils/urlHelper';
 import * as AppReleaseActionTypes from './../appReleaseActionTypes';
 import AppReleaseActionDispatcher from './../appReleaseDispatcher';
+import { IRelease, BuildResult, BuildStatus } from '../types';
+
+const mockResults: IRelease[] = [
```

Comment:
should we merge with the mock to the branch? Maybe comment out?

Reply:
Commented out with last commit.